### DISCUSSION TITLE
Can I access the API with an Authorization header?

### DISCUSSION THREAD

**Discussion by emregency** [directus/directus#3327] [Archive]
I tried to create a user with a token instead of a password. I was hoping to pass the token along with an Authorization header to access the API programmatically. That way I would prevent to first login and then execute my action. Does that make sense?

**Comment by rijkvanzanten** [comment]
It does make sense, and works the way you describe 👍 

You can pass the token in a query param `?access_token=abc` or Authorization header `Authorization: Bearer abc`

**Reply by emregency** [reply]
Awesome! I missed the Bearer part. Thanks 🙏

**Reply by cynova** [reply]
Thank you @rijkvanzanten for implementing this. Having to login and keep a session was really annoying in V8 for serverside clients, and i love the way it's done now. Excellent work as always

**Reply by emregency** [reply]
Hey guys, 

I am still having difficulties with this one 🙃. I created a user with a token (without a password) and I am just using its token as a Bearer token in the authorization header.

I am getting an invalid credentials error. 

What am I doing wrong exactly?

**Reply by emregency** [reply]
Never mind, special characters were the problems 😇

**Comment by rijkvanzanten** [comment]
See https://docs.directus.io/reference/api/authentication/ 🙂
